src/com/mraof/minestuck/inventory/captchalouge/CaptchaDeckHandler.java

```diff
@@ -254,6 +261,66 @@ else if(card1 && card2)
 		
 	}
 	
+	public static void captchalougeInventoryItem (EntityPlayerMP player) {
+		GuiScreen guiScreen = Minecraft.getMinecraft().currentScreen;
```

Comment:
This approach will not work as guis aren't available on the server side, while this function clearly is on the server side